src/main/java/org/fogbowcloud/app/jes/arrebol/models/ArrebolTask.java

```diff
@@ -0,0 +1,24 @@
+package org.fogbowcloud.app.jes.arrebol.models;
+
+
+public class ArrebolTask {
```

Comment:
You need to update the Arrebol Task attributes, now Arrebol Task Id is not equal to Iguassu Task Id.

As you can check [here](https://github.com/ufcg-lsd/arrebol/blob/scheduler/src/main/java/org/fogbowcloud/arrebol/models/task/Task.java), Arrebol Tasks have a [TaskSpec](https://github.com/ufcg-lsd/arrebol/blob/scheduler/src/main/java/org/fogbowcloud/arrebol/models/task/TaskSpec.java)  that contains the Iguassu Task Id, so, for now, you need to add the TaskSpec representation in Iguassu to associate the Task representation in both projects.

Reply:
Ok. Done.